tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -714,7 +749,7 @@ func (c *Reconciler) updatePipelineRunStatus(ctx context.Context, iterationEleme
 	if err != nil {
 		return 0, nil, nil, fmt.Errorf("could not list PipelineRuns %#v", err)
 	}
-	if pipelineRuns == nil || len(pipelineRuns) == 0 {
+	if len(pipelineRuns) == 0 {
```

Comment:
What error will you get for len(pipelineRuns) if pipelineRuns is nil? should you also handle this case?

Reply:
As per this, https://go.dev/tour/moretypes/12 len() on Nil slices in golang return 0. So additional check of slice == nil was redundant.